packages/js-sdk/src/User.d.ts

```diff
@@ -4,46 +4,59 @@ import type { Transaction } from 'web3';
 import type { Flow } from './ConstantFlowAgreementV1Helper';
 import type { Subscription } from './InstantDistributionAgreementV1Helper';
 
-export type DetailsType = {
+// params options types
+export interface UserOptions {
+    sf: Framework;
+    address: string;
+    token: string;
+    options?: any;
```

Comment:
this can be removed, if you look in the `User.js` file, we removed this because it was not being used for anything.

Reply:
Removed the 'options' from UserOptions, but I left the interface, as it types the params of the User constructor: Framework, address, and token (useful for external ts packages).